libavfilter/f_ebur128.c

```diff
@@ -137,6 +137,10 @@ typedef struct EBUR128Context {
     int target;                     ///< target level in LUFS used to set relative zero LU in visualization
     int gauge_type;                 ///< whether gauge shows momentary or short
     int scale;                      ///< display scale type of statistics
+    int gate_measurement;           ///< whether or not to use the gated measurement
+    double *ch_raw_integrated_power; ///< each channels's power
+    double raw_integrated_power_sum; ///< total powers since beginning
+    long raw_power_count;            ///< total sample count
```

Comment:
I think it's more accurate to change this from `raw_power_count` to `ungated_count`

Reply:
Additionally change the comment to `///< Count of number of ungated samples since beginning of measuring`